buttercup/cogs/rules.py

```diff
@@ -78,6 +120,73 @@ async def _rules(self, ctx: SlashContext, subreddit: str) -> None:
             embed=embed,
         )
 
+    @cog_ext.cog_slash(
+        name="pirules",
+        description="Get the rules of the specified subreddit "
+        "regarding personal information.",
+        options=[
+            create_option(
+                name="subreddit",
+                description="The subreddit to get the PI rules of.",
+                option_type=3,
+                required=True,
+            )
+        ],
+    )
+    async def _pi_rules(self, ctx: SlashContext, subreddit: str) -> None:
+        """Get the rules of the specified subreddit regarding personal information."""

```

Comment:
I feel like this method and the `_rules` method share quite a lot of code. Do you think that there is a way to combine the common functionality of the two?

Reply:
Yea fair point. I extracted some common functionality now.